### PR TITLE
Modify clock script to just run the jobs instead of the schedule.

### PR DESCRIPTION
**Pull request by pmclanahan**
Basically the same as #3295, but just runs all the jobs with the enhanced logging insetad of scheduling them. This will make it very easy to enable the scheduler process when deis is ready.

### CODE REVIEW

**Comment by pmclanahan** [comment]
@jgmize r?

File: scripts/clock.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+
+@scheduled_job('interval', hours=1)
+def job_ical_feeds():
+    update_ical_feeds()
+    cleanup_ical_events()
+
+
+@scheduled_job('interval', hours=1)
+def job_update_firefox_os_feeds():
```
Comment (pmclanahan) [inline scripts/clock.py]: Note: `update_firefox_os_feeds` and `update_tableau_data` were left out of the previous version. Not sure we even have access to the tableau DB from AWS (likely we do not).

**Comment by pmclanahan** [comment]
Are we not happy with this approach? I know @bensternthal wanted to get this in and get some alerts setup to watch for these log messages. Are you not happy about adding the dependency before we get the ability to run a separate process @jgmize? We could always create a separate app that would run the clock proc for real and give it the same settings for DB and whatnot, it'd just be a pain.

**Comment by jgmize** [comment]
> Are we not happy with this approach? I know @bensternthal wanted to get this in and get some alerts setup to watch for these log messages. 

Sorry, @pmclanahan, I'm not sure this will accomplish that goal-- I believe the outputs from the print statements will return to the deis client and won't show up in papertrail.

> Are you not happy about adding the dependency before we get the ability to run a separate process @jgmize? 

Now that you mention it, I'm not that excited about the fact that this PR adds a fairly large dependency and then ends up not actually using any of its functionality. However, based on my reading of https://github.com/deis/deis/pull/4606 the issue is entirely on the deis client side, so we could try deploying a bedrock app with a procfile using either the older python version of the deis client, or if we were feeling adventurous we could build a deis client based on that PR and try using it.

**Comment by pmclanahan** [comment]
ADVENTURE TIME!

**Comment by pmclanahan** [comment]
@jgmize looks like said deis PR was merged. Proceed how you will.

**Comment by pmclanahan** [comment]
@jgmize Just following up on some PRs. Have you tried the new deis client that's supposed to fix the Procfile issue? Should I just close this one?

**Comment by jgmize** [comment]
@pmclanahan I haven't had a chance to build a deis client binary based on master to test it, but that fix should be in the 1.12 version of the client, which should be released a week from today according to http://docs.deis.io/en/latest/roadmap/schedule/#minor-releases

**Comment by pmclanahan** [comment]
Closing in favor of #3960

**Pull request closed without merging**